indexer/test/advertisement-walker.test.js

```diff
@@ -336,7 +336,7 @@ describe('processNextAdvertisement', () => {
     })
   })
 
-  it('handles timeout errors and explains the problem in the status', async () => {
```

Comment:
This test is failing on my machine, and I could not find any quick fix. I'd like to investigate the problem after I land this PR.